dev-ruby/facter/facter-3.11.3-r1.ebuild

```diff
@@ -66,12 +70,18 @@ src_prepare() {
 	cmake-utils_src_prepare
 }
 
+each_ruby_configure() {
+	# hack for correct calculation of relative path from facter.rb to
+	# libfacter.so
+	my_ruby_sitelibdir=$(ruby_rbconfig_value 'sitelibdir')
+}
+
 src_configure() {
+	ruby-ng_src_configure
+
 	local mycmakeargs=(
 		-DCMAKE_VERBOSE_MAKEFILE=ON
-		-DCMAKE_BUILD_TYPE=None
```

Comment:
why'd you get rid of the build type? and install prefix? and blkid location?

Reply:
CMAKE_BUILD_TYPE and CMAKE_INSTALL_PREFIX are passed by cmake-utils.eclass. Do we really need to override them? BLKID_LIBRARY is not used by build system for a long time:
https://github.com/puppetlabs/facter/commit/364b947413fd1ba7eb445fc7fde96cb96cb60d72#diff-dd2044c7974dda8071b268f0652e866e